src/main/java/com/microsoft/sqlserver/jdbc/SQLServerBulkCSVFileRecord.java

```diff
@@ -516,4 +523,58 @@ public boolean next() throws SQLServerException {
         }
         return (null != currentLine);
     }
+
+    /**
+     * Returns whether the rules to escape delimiters are used.
+     *
+     * @return true if the rules are used, false otherwise.
+     */
+    public boolean isEscapeColumnDelimitersCSV() {
```

Comment:
I think we should rename this to getEscapeColumnDelimitersCSV, for the setter/getter convention.

Reply:
I named it this way mainly to keep it consistent with SQLServerBulkCopyOptions and SQLServerBulkRecord.